core/js/src/git_fields.ts

```diff
@@ -10,12 +13,27 @@ export interface RepoInfo {
   git_diff?: string;
 }
 
-export type GitFields = Array<keyof RepoInfo>;
-export type CollectMetadata = "all" | "none" | "some";
-export type GitMetadataSettings = {
-  collect: CollectMetadata;
-  fields?: GitFields;
-};
+export const gitFieldsSchema = z.enum([
```

Comment:
These are exactly the keys of `repoInfoSchema`. Is it possible to get them from there?